helpers/middleman/menu.rb

```diff
@@ -0,0 +1,265 @@
+load File::expand_path('./locale.rb', __dir__)
```

Comment:
I just copied what I had before when I refactored all this Ruby code from in-line `html.erb` spaghetti 🍝 , but looking into it:
 - `require_relative` does not hot reload in the "index file" at `helpers/middleman/helpers.rb`, whereas `load` does.
 - `require_relative` does work in the child files, eg. in this file, but I think it's because the "index file" is re-loading them all…
 - So, I'll stick with `load` to be consistent.
 - Added a comment in the "index file" in 18aa4807 so I don't have to look into this again 😃 

_If you don't have hot-reloading, you have to kill the server and restart, reloading Contentful, etc…takes a long time.  Given we'll make tiny changes to this file and see them on the page, the 5 second hot reload is much more useful than 30-60s._